docker.py

```diff
@@ -888,6 +888,25 @@ def get_scipy_choices():
     protobuf='3.3.0',
 )
 
+# cuSPARSELt
+cusparselt_base = '''
+RUN tmpdir=`mktemp -d` && \\
+    cd $tmpdir && \\
+    curl -sL -o {cusparselt}.tar.gz https://developer.download.nvidia.com/compute/libcusparse-lt/{cusparselt_ver}/local_installers/{cusparselt}.tar.gz && \\
+    tar -xzf {cusparselt}.tar.gz && \\
+    cd libcusparse_lt && \\
+    cp ./include/* /usr/local/cuda/include && \\
+    cp ./lib64/* /usr/local/cuda/lib64 && \\
```

Comment:
Please use `-a` to preserve file attributes. (it will also make the script runnable if directory is added in future cusparselt release)

```suggestion
    cp -a ./include/* /usr/local/cuda/include && \\
    cp -a ./lib64/* /usr/local/cuda/lib64 && \\
```